javascript-modal/main.js

```diff
@@ -0,0 +1,21 @@
+function openDialogue(event) {
+  var confirm = document.querySelector('#confirm');
+  confirm.style.display = 'block';
+  var shadey = document.querySelector('#shade');
+  shadey.style.display = 'block';
+}
+
+function closeDialogue(event) {
+  var confirm = document.querySelector('#confirm');
+  confirm.style.display = 'none';
+  var shadey = document.querySelector('#shade');
+  shadey.style.display = 'none';
+
+}
+window.addEventListener('load', function () {
```

Comment:
You don't need this event handler, you can just do the work of this function inside the js file directly, you want it to happen as soon as this file loads.